Content.Server/_Starlight/Antags/Abductor/AbductorSystem.Actions.cs

```diff
@@ -0,0 +1,247 @@
+        //ev.Handled = true;
+    //}
+
+    //private void OnDoAfterAbductorScientistSendYourself(Entity<AbductorScientistComponent> ent, ref AbductorSendYourselfDoAfterEvent args)
+    //{
+        //OnDoAfterSendYourself(ent, args);
+    //}
+
+    //private void OnDoAfterAbductorAgentSendYourself(Entity<AbductorAgentComponent> ent, ref AbductorSendYourselfDoAfterEvent args)
+    //{
+        //OnDoAfterSendYourself(ent, args);
+    //}
+
+    //private void OnDoAfterSendYourself(EntityUid ent, AbductorSendYourselfDoAfterEvent args)
+    //{
+        //_color.RaiseEffect(Color.FromHex("#BA0099"), new List<EntityUid>(1) { ent }, Filter.Pvs(ent, entityManager: EntityManager));
+        //if (_pullingSystem.IsPulling(ent))
+        //{
+            //if (!TryComp<PullerComponent>(ent, out var pullerComp)
+                //|| pullerComp.Pulling == null
+                //|| !TryComp<PullableComponent>(pullerComp.Pulling.Value, out var pullableComp)
+                //|| !_pullingSystem.TryStopPull(pullerComp.Pulling.Value, pullableComp)) return;
+        //}
+
+        //if (_pullingSystem.IsPulled(ent))
+        //{
+            //if (!TryComp<PullableComponent>(ent, out var pullableComp)
+                //|| !_pullingSystem.TryStopPull(ent, pullableComp)) return;
+        //}
+        //_xformSys.SetCoordinates(ent, GetCoordinates(args.TargetCoordinates));
+        //OnCameraExit(ent);
+    //}
+
+    //private void OnGizmoMark(GizmoMarkEvent ev)
+    //{
+        //if (!HasComp<AbductorComponent>(ev.Target))
+            //return;
+
+        //if (!_inv.TryGetSlotContainer(ev.Performer, "pocket1", out var pocket1, out _) || !_inv.TryGetSlotContainer(ev.Performer, "pocket2", out var pocket2, out _))
+            //return;
+
+        //var pocket1PossibleGizmo = pocket1.ContainedEntity;
+        //var pocket2PossibleGizmo = pocket2.ContainedEntity;
+
+        //if (TryComp<AbductorGizmoComponent>(pocket1PossibleGizmo, out var pocket1Gizmo))
+            //pocket1Gizmo.Target = GetNetEntity(ev.Target);
+        //else if (TryComp<AbductorGizmoComponent>(pocket2PossibleGizmo, out var pocket2Gizmo))
+            //pocket2Gizmo.Target = GetNetEntity(ev.Target);
+
+    //}
+
+    private void OnExit(ExitConsoleEvent ev) => OnCameraExit(ev.Performer);
+
+    private void AddActions(AbductorBeaconChosenBuiMsg args)
+    {
+        EnsureComp<AbductorsAbilitiesComponent>(args.Actor, out var comp);
+        comp.HiddenActions = _actions.HideActions(args.Actor);
+        _actions.AddAction(args.Actor, ref comp.ExitConsole, _exitAction);
+        //_actions.AddAction(args.Actor, ref comp.SendYourself, _sendYourself);
+        //_actions.AddAction(args.Actor, ref comp.GizmoMark, _gizmoMark);
+    }
+    private void RemoveActions(EntityUid actor)
+    {
+        EnsureComp<AbductorsAbilitiesComponent>(actor, out var comp);
+        if (comp.ExitConsole is not null)
+            _actions.RemoveAction(actor, comp.ExitConsole);
```

Comment:
```
if (comp.ExitConsole is { } exitConsole)
    _actions.RemoveAction(actor, exitConsole);
```
regardless, this should be refactored per the comment in `AbductorsComponent.cs`